ceph.spec.in

```diff
@@ -1183,6 +1186,9 @@ fi
 %{_mandir}/man8/ceph-clsinfo.8*
 %{_mandir}/man8/ceph-disk.8*
 %{_mandir}/man8/ceph-osd.8*
+%if 0%{?rhel}
+/etc/cron.hourly/subman.py
```

Comment:
@dachary: the issue is that we do not package the pyc and pyo files that python creates, here. That would be covered if added * at the end of this line. However, I'm not sure how well cron handles these pyc/pyo files. It would probably be better to remove the .py extension so that the files don't get created at all.